README.md

```diff
@@ -1,8 +1,32 @@
 # SymPy Paper
 
-[![Join the chat at https://gitter.im/asmeurer/sympy-paper](https://badges.gitter.im/asmeurer/sympy-paper.svg)](https://gitter.im/asmeurer/sympy-paper?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+This is the source repository for the paper:
+
+> SymPy Authors, "SymPy: Symbolic Computing in Python", 2016.
+
+This repository contains or links to all of the information needed to reproduce
+the results in the paper.
+
+The latest rendered version of the PDF can be viewed via the
+[ShareLaTeX](http://sharelatex.com) CI system:
 
 [![PDF Status](https://www.sharelatex.com/github/repos/asmeurer/sympy-paper/builds/latest/badge.svg)](https://www.sharelatex.com/github/repos/asmeurer/sympy-paper/builds/latest/output.pdf)
 
+# Contributing
+
+Join our chat room to discuss contributions:
+
+[![Join the chat at https://gitter.im/asmeurer/sympy-paper](https://badges.gitter.im/asmeurer/sympy-paper.svg)](https://gitter.im/asmeurer/sympy-paper?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+
+# License
+
+The contents of this repository is licensed under the [Creative Commons
+Attribution 4.0 International
+License](http://creativecommons.org/licenses/by/4.0).
+
+[![CCBY](https://i.creativecommons.org/l/by/4.0/80x15.png)](http://creativecommons.org/licenses/by/4.0)
```

Comment:
Debian webpages are dual licensed under MIT and GPL:
https://www.debian.org/license

Sent from my mobile phone.
On Apr 18, 2016 6:35 PM, "Aaron Meurer" <notifications@github.com> wrote:

> In README.md
> <https://github.com/sympy/sympy-paper/pull/57#discussion_r60158052>:
>
> >
> >  [![PDF Status](https://www.sharelatex.com/github/repos/asmeurer/sympy-paper/builds/latest/badge.svg)](https://www.sharelatex.com/github/repos/asmeurer/sympy-paper/builds/latest/output.pdf)
> >
> > +# Contributing
> > +
> > +Join our chat room to discuss contributions:
> > +
> > +[![Join the chat at https://gitter.im/asmeurer/sympy-paper](https://badges.gitter.im/asmeurer/sympy-paper.svg)](https://gitter.im/asmeurer/sympy-paper?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
> > +
> > +# License
> > +
> > +The contents of this repository is licensed under the [Creative Commons
> > +Attribution 4.0 International
> > +License](http://creativecommons.org/licenses/by/4.0).
> > +
> > +[![CCBY](https://i.creativecommons.org/l/by/4.0/80x15.png)](http://creativecommons.org/licenses/by/4.0)
>
> CC-BY isn't listed on the OSI site
> <https://opensource.org/licenses/alphabetical>, but interestingly enough,
> the contents of the site itself are licensed CC-BY 4.0.
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/sympy/sympy-paper/pull/57/files/a336236f4a810f18b59d0b4afed4c03a5a5d5cd0#r60158052>
>